tests/minio-container/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM minio/minio
+COPY scripts/startup.sh /startup.sh
+COPY policies /policies
+COPY files /files
+RUN chmod +x /startup.sh
```

Comment:
This should not be necessary if you would checkin the file already with the executable bit set.
```suggestion
```